docs/ui-components/callouts.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: default
+title: Callouts
+parent: UI Components
+nav_order: 7
+---
+
+# Callouts
+
+Markdown does not include support for callouts. However, you can style text as a callout using a Markdown extension supported by kramdown: [*block IALs*](https://kramdown.gettalong.org/quickref.html#block-attributes). 
+
+Common kinds of callouts include `admonition`, `attention`, `caution`, `danger`, `error`, `hint`, `important`, `note`, `tip`, and `warning`.
+
+When you have [configured]({{ site.baseurl }}{% link docs/configuration.md %}#callouts) the  `color` and (optional) `title` for a callout, you can apply it to a paragraph, or to a block quote with several paragraphs, as illustrated below.[^postfix]
+
+[^postfix]:
+    You can put the callout markup either before or after its content.
+
+#### An untitled callout
+{: .no_toc }
+
+```markdown
+{: .highlight }
+A paragraph
+```
```

Comment:
What are your thoughts on including some custom styles so that users can see callouts in action on this page? Perhaps overriding some variables just on this page?

(this can be deferred until later)

Reply:
To show callouts in action would be good practice, and consistent with the illustrations on the pages for buttons, labels, etc.

However, the uniform association of callouts with particular colors throughout a website currently uses config files (in contrast to the ad hoc associations of buttons and labels with colors). I'm not sure how easy it would be to simulate such configuration on a particular page.

Let's defer it.

Reply:
Gotcha, sounds good. Backlog it is!

Reply:
Assuming that issue #171 and PR #466 will be closed when this PR is merged, I think the best will be if I submit a new issue for adding examples of the appearance of callouts to the docs. But it doesn't make sense to submit it before v0.4.0 has been released…